### PR TITLE
modules/keymaps: doc: add backticks around special arguments

### PR DESCRIPTION
**Pull request by iSma**
As mentioned here: https://github.com/nix-community/nixvim/pull/1565#discussion_r1614916649

### CODE REVIEW

File: lib/keymap-helpers.nix [hunk 1]
```diff
@@ -7,23 +7,23 @@ with lib;
...
-    script = nixvimOptions.defaultNullOpts.mkBool false "Equivalent to adding <script> to a map.";
+    script = nixvimOptions.defaultNullOpts.mkBool false "Equivalent to adding `<script>` to a map.";
 
-    expr = nixvimOptions.defaultNullOpts.mkBool false "Means that the action is actually an expression. Equivalent to adding <expr> to a map.";
+    expr = nixvimOptions.defaultNullOpts.mkBool false "Means that the action is actually an expression. Equivalent to adding `<expr>` to a map.";
 
-    unique = nixvimOptions.defaultNullOpts.mkBool false "Whether to fail if the map is already defined. Equivalent to adding <unique> to a map.";
+    unique = nixvimOptions.defaultNullOpts.mkBool false "Whether to fail if the map is already defined. Equivalent to adding `<unique>` to a map.";
 
     noremap = nixvimOptions.defaultNullOpts.mkBool true "Whether to use the 'noremap' variant of the command, ignoring any custom mappings on the defined action. It is highly advised to keep this on, which is the default.";
```
Comment (MattSturgeon) [inline lib/keymap-helpers.nix:20]:
```suggestion
    noremap = nixvimOptions.defaultNullOpts.mkBool true "Whether to use the `noremap` variant of the command, ignoring any custom mappings on the defined action. It is highly advised to keep this on, which is the default.";
```

File: lib/keymap-helpers.nix [hunk 2]
```diff
@@ -7,23 +7,23 @@ with lib;
...
 
     remap = nixvimOptions.defaultNullOpts.mkBool false "Make the mapping recursive. Inverses \"noremap\"";
```
Comment (MattSturgeon) [inline lib/keymap-helpers.nix:22]:
```suggestion
    remap = nixvimOptions.defaultNullOpts.mkBool false "Make the mapping recursive. Inverses `noremap`.";
```

**Review by MattSturgeon — COMMENTED** [review]
Great! Personally, I'd go further and also do the `noremap` option

**Comment by iSma** [comment]
> Great! Personally, I'd go further and also do the `noremap` option

Done :+1:

**Review by MattSturgeon — APPROVED** [review]
Excellent! Thanks for doing this.

**Review by traxys — APPROVED** [review]

**Pull request merged**